src/app/layout.tsx

```diff
@@ -71,6 +74,10 @@ export default function RootLayout({
 }>) {
   return (
     <html lang='en' className={`${helveticaNeue.variable} font-sans`}>
+      <Script
+        src='https://cdnjs.cloudflare.com/ajax/libs/html2canvas/1.4.1/html2canvas.min.js'
```

Comment:
Question: why did you add this, what is it for?

Reply:
I've been digging into ways to improve our Core Web Vitals, especially TBT, which seems to be our main bottleneck right now. I found few suggestions about using html2canvas to potentially help with rendering consistency across different screen sizes, but unfortunately, it didn't make a positive difference in our case.

I completely forgot to remove the html2canvas code after testing it out ... gonna remove this